packages/core/src/awsService/ec2/explorer/ec2InstanceNode.ts

```diff
@@ -67,6 +76,10 @@ export class Ec2InstanceNode extends AWSTreeNodeBase implements AWSResourceNode
         }
     }
 
+    public getStatus(): string {
+        return this.instance.status!
```

Comment:
why would it be undefined? can we avoid that or make undefined a return type?

Reply:
also can we type status? maybe to `typeof Ec2Instance.status`

Reply:
> why would it be undefined? can we avoid that or make undefined a return type?

Yeah, I realized I did this in a few places. I refactored the code to use an interface `SafeEc2Instance` that won't let me do that anymore.